Source/environment.py

```diff
@@ -0,0 +1,137 @@
+# Author: Siena Okuno
+# Project: Interactive Programming
+# Date: 10.30.2017
+# Description: Creates GUI for player and posts events when a button is clicked
+
+import pygame
+import os
+import sys
+
+def initialize(descriptions):
+    """ Initializes graphic screen.
+    """
+    pygame.init()
+    screen = pygame.display.set_mode((1000, 700))
+    pygame.font.init()
+    font = pygame.font.Font(None,26)
+    pygame.mouse.set_pos([500,350])
+    pygame.mouse.set_visible(True)
+    text = setUpE(screen, font, descriptions)
+    return [screen, text]
+
+def setUpE(screen, font, descriptions):
+    '''This code is to load the basic backscreen for the program.
+    setUpE is NOT to be run whenever the user clicks something, just at beginning.'''
+    #loading/scaling images (loads as S___ for start, scaled as F___ for final)
+    Sbee_Icon = pygame.image.load('Source/Course_Icons/bees.png')
+    Fbee_Icon = pygame.transform.scale(Sbee_Icon, (108, 100))
+    SENM_Icon = pygame.image.load('Source/Course_Icons/elec.png')
+    FENM_Icon = pygame.transform.scale(SENM_Icon, (108, 100))
+    SFOCS_Icon = pygame.image.load('Source/Course_Icons/focs.png')
+    FFOCS_Icon = pygame.transform.scale(SFOCS_Icon, (108, 100))
+    SFunRobo_Icon = pygame.image.load('Source/Course_Icons/robo.png')
+    FFunRobo_Icon = pygame.transform.scale(SFunRobo_Icon, (108, 100))
+    SMatSci_Icon = pygame.image.load('Source/Course_Icons/mats.png')
+    FMatSci_Icon = pygame.transform.scale(SMatSci_Icon, (108, 100))
+    SMechProto_Icon = pygame.image.load('Source/Course_Icons/mech.png')
+    FMechProto_Icon = pygame.transform.scale(SMechProto_Icon, (108, 100))
+    #basic background
```

Comment:
To take this to the next level, consider replacing these separate variables by a list.